integer_validator(self, name, value)
======================

base_geometry fn
-------------------

>>> BaseGeometry = __import__('7-base_geometry').BaseGeometry
>>> bg = BaseGeometry()

first check:

>>> bg.integer_validator("yass", "ha")
Traceback (most recent call last):
TypeError: yass must be an integer


second check

>>> bg.integer_validator("yass", [99])
Traceback (most recent call last):
TypeError: yass must be an integer

third check
>>> bg.integer_validator("yass", 9.999)
Traceback (most recent call last):
TypeError: yass must be an integer



fourth check

>>> bg.integer_validator() 
Traceback (most recent call last):
TypeError: integer_validator() missing 2 required positional arguments: 'name' and 'value'

fifth check
>>> bg.integer_validator("yass", 77, 3)
Traceback (most recent call last):
TypeError: integer_validator() takes 3 positional arguments but 4 were given

sixth check:
>>> bg.integer_validator(None) 
Traceback (most recent call last):
TypeError: integer_validator() missing 1 required positional argument: 'value'